### DISCUSSION TITLE
platform/storage/dataset

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#783] [Comments]
# platform/storage/dataset

Store and export web scraping, crawling or data processing job results. Learn how to access and manage datasets in Apify Console or via API.

https://docs.apify.com/platform/storage/dataset

**Comment by max4ever** [comment]
The API get dataset returns itemCount https://docs.apify.com/api/v2/#/reference/datasets/dataset/get-dataset, but I can't find this in the javascript SDK. Is there a way to get the length without looping over all items ?

**Reply by metalwarrior665** [reply]
@max4ever 
```js
const { itemCount } = await Actor.apifyClient.dataset(datasetId).get();
```

**Reply by max4ever** [reply]
Unfortunately this didn't work, `await Actor.apifyClient...` returns `undefined`

**Reply by metalwarrior665** [reply]
That suggests the dataset doesn't exist. Can you share your code?

**Reply by max4ever** [reply]
thanks, I put the full name of the dataset and it worked 👍 

I was using `Actor.openDataset('datasetId')`, but with apifyClient I have to write `Actor.apifyClient.dataset('accountName/datasetId')`
